From 4b7c661d08d27f52a6ffda504e3d4f943e0d694c Mon Sep 17 00:00:00 2001
From: Nic Costa <nic.costa@gmail.com>
Date: Sat, 9 Mar 2019 15:57:37 -0600
Subject: [PATCH] fix compile error when building for Release

This fixes the following compilation error when building
in a Yocto environment:

  warning _FORTIFY_SOURCE requires compiling with optimization (-O)
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a7b9939..1f39db4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -52,7 +52,7 @@ include (cmake/edge_include_directories.cmake)
 include(git_details.cmake)
 
 ADD_DEFINITIONS(-D__STDC_FORMAT_MACROS)
-if (NOT (CMAKE_BUILD_TYPE STREQUAL release))
+if (NOT (CMAKE_BUILD_TYPE STREQUAL Release))
     SET (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O0")
     SET (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O0")
 endif ()
-- 
2.7.4

